rocky/account/migrations/0003_alter_katuser_id.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.1 on 2023-05-19 15:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("account", "0002_remove_first_last_name"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="katuser",
+            name="id",
+            field=models.AutoField(primary_key=True, serialize=False, verbose_name="ID"),
```

Comment:
I'm not sure why this migration was created, the only difference seems to be that `serialize=False` was added. I couldn't find anything about this in the Django 4.0-4.2 release notes.

Is there any downside in keeping this migration?

Reply:
As far as I understand it `primary_key=True` causes `serialize=False`. If you normally create migrations you get `primary_key=True` and `serialize=False`, but the katuser is created by a custom written migration to migrate off the standard Django User model and didn't have `serialize=False` . I think it's just some internal thing that works differently in 4.2 which causes the migration code to think something has changed while it didn't think that before.

There is no downside in keeping the migration except that every migration has a tiny cost when running `manage.py migrate` etc. The other option is to just add `serialize=False` in the 0001 migration, given that was already implicitly the case because of `primary_key=True` adding that shouldn't cause any problems.

Reply:
Thanks for the insight, I think it's likely to be caused by the [migrations autodetector changes](https://docs.djangoproject.com/en/4.2/releases/4.0/#migrations-autodetector-changes) in Django 4.0.

I've changed the initial migration like you suggested so we can avoid the additional migration.